recipes/r-loom/meta.yaml

```diff
@@ -8,15 +8,15 @@ source:
   sha256: 7ad37bee3f91cc7d0cfb7e31b2de33e4bf7dd03ed2205732c34f150c97faa652 
 
 build:
-  noarch: generic
-  number: 0
+  number: 1
   noarch: generic
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
   host:
+    - icu >=64
```

Comment:
mh, this is not good. The problem that is causing the problem should be fixed instead.